ts/components/cie/CieNotSupported.tsx

```diff
@@ -1,80 +1,101 @@
-  hasCieNFCFeature: boolean;
-};
+const CieNotSupported = () => {
+  const hasApiLevelSupport = useIOSelector(hasApiLevelSupportSelector);
+  const hasCieApiLevelSupport = pot.getOrElse(hasApiLevelSupport, false);
+  const hasNFCFeature = useIOSelector(hasNFCFeatureSelector);
+  const hasCieNFCFeature = pot.getOrElse(hasNFCFeature, false);
+  const accessibilityFirstFocuseViewRef = useRef<View>(null);
+  const navigation = useNavigation();
 
-const ICON_SIZE = 24;
-const okColor: IOColors = "green";
-const koColor: IOColors = "red";
-const okIcon: IOIcons = "ok";
-const koIcon: IOIcons = "errorFilled";
-const markDownElements = 2;
-const CieNotSupported: React.FunctionComponent<Props> = props => {
-  const [markdownLoaded, setMarkdownLoaded] = useState(0);
-  const handleMarkdownLoaded = () => setMarkdownLoaded(s => s + 1);
-  return (
-    <React.Fragment>
-      <LegacyMarkdown onLoadEnd={handleMarkdownLoaded}>
-        {I18n.t("authentication.landing.cie_unsupported.body")}
-      </LegacyMarkdown>
+  useFocusEffect(() => setAccessibilityFocus(accessibilityFirstFocuseViewRef));
 
-      {Platform.OS === "android" && (
-        <React.Fragment>
-          <VSpacer size={16} />
-          <LegacyMarkdown onLoadEnd={handleMarkdownLoaded}>
-            {I18n.t("authentication.landing.cie_unsupported.android_desc")}
-          </LegacyMarkdown>
-          <VSpacer size={40} />
-          {markdownLoaded === markDownElements && (
-            <List>
-              <ListItem>
-                <Icon
-                  size={ICON_SIZE}
-                  name={props.hasCieApiLevelSupport ? okIcon : koIcon}
-                  color={props.hasCieApiLevelSupport ? okColor : koColor}
-                />
-                <HSpacer size={8} />
-                <Body>
-                  {I18n.t(
-                    "authentication.landing.cie_unsupported.os_version_unsupported"
-                  )}
-                </Body>
-              </ListItem>
-              <ListItem>
-                <View style={{ alignItems: "flex-start" }}>
-                  <Icon
-                    size={ICON_SIZE}
-                    name={props.hasCieNFCFeature ? okIcon : koIcon}
-                    color={props.hasCieNFCFeature ? okColor : koColor}
-                  />
-                </View>
-                <HSpacer size={8} />
-                <Body>
-                  {I18n.t(
-                    "authentication.landing.cie_unsupported.nfc_incompatible"
-                  )}
-                </Body>
-              </ListItem>
-            </List>
+  return (
+    <GradientScrollView
+      testID="container-test"
+      primaryActionProps={{
+        label: I18n.t("authentication.landing.cie_unsupported.button"),
+        accessibilityLabel: I18n.t(
+          "authentication.landing.cie_unsupported.button"
+        ),
+        onPress: () => navigation.goBack(),
+        testID: "close-button"
+      }}
+    >
+      <SafeAreaView>
```

Comment:
Isn't the safe area already taken into consideration by the `GradientScrollView`?

Reply:
Yess! Thanks!